<ui version="4.0" >
 <class>ThumbnailViewPanel</class>
 <widget class="QWidget" name="ThumbnailViewPanel" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>434</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <property name="spacing" >
    <number>0</number>
   </property>
   <item>
    <widget class="Gwenview::ThumbnailView" name="mThumbnailView" />
   </item>
   <item>
    <widget class="QWidget" native="1" name="mFilterBar" >
     <layout class="QHBoxLayout" name="horizontalLayout" >
      <property name="leftMargin" >
       <number>0</number>
      </property>
      <property name="topMargin" >
       <number>9</number>
      </property>
      <property name="rightMargin" >
       <number>0</number>
      </property>
      <property name="bottomMargin" >
       <number>9</number>
      </property>
      <item>
       <widget class="QLabel" name="label_2" >
        <property name="text" >
         <string>Minimum Rating:</string>
        </property>
        <property name="buddy" >
         <cstring>mRatingSlider</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="mRatingSlider" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximum" >
         <number>5</number>
        </property>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition" >
         <enum>QSlider::TicksBelow</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>Filter:</string>
        </property>
        <property name="buddy" >
         <cstring>mFilterEdit</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="KLineEdit" name="mFilterEdit" >
        <property name="clickMessage" >
         <string>Enter search term here...</string>
        </property>
        <property name="showClearButton" stdset="0" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" >
     <item>
      <widget class="QToolButton" name="mFilterBarButton" >
       <property name="text" >
        <string>...</string>
       </property>
       <property name="autoRaise" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QSlider" name="mThumbnailSlider" >
       <property name="maximumSize" >
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="minimum" >
        <number>40</number>
       </property>
       <property name="maximum" >
        <number>256</number>
       </property>
       <property name="value" >
        <number>128</number>
       </property>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>Gwenview::ThumbnailView</class>
   <extends>QListView</extends>
   <header location="global" >lib/thumbnailview/thumbnailview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
